elephant/csd.py

```diff
@@ -1,3 +1,357 @@
 #!/usr/bin/env python
-'''Current source density estimation methods from extracellular potentials'''
-from .icsd.icsd import estimate_csd
+"""This script is used to generate Current Source Density Estimates
+This was written by :
+Chaitanya Chintaluri,
```

Comment:
Can you put here also Michał and Espen. This is the module which is the entry point to the whole CSD analysis, so most people will have a look here. A short sentence indicating who did what would be sufficient. You could write `contributors to the current source density module: ...`